src/components/Lists/ListBase/index.tsx

```diff
@@ -0,0 +1,17 @@
+import {
+    List as MaterialList,
+    ListProps as MaterialListProps,
+    Box as MaterialBox,
+} from "@material-ui/core";
+
+export interface ListBaseProps extends MaterialListProps {}
+
+const ListBase = (props: ListBaseProps) => {
+    return (
+        <MaterialBox width={320} color="secondary.main" bgcolor="primary.main">
+            <MaterialList {...props} />
+        </MaterialBox>
+    );
+};

```

Comment:
We will be using list for a lot of other purposes. If you want to set width, please make a different component